### DISCUSSION TITLE
Nothing happens - What am I doing wrong?

### DISCUSSION THREAD

**Discussion by Schwobi** [tteck/Proxmox#2381] [Help Needed] [answered]
As soon as a copy a command ("bash..") insert it and hit ENTER -> nothing happens. What am I doing wrong? What am i missing? 
Bash seems to be installed.
![grafik](https://github.com/tteck/Proxmox/assets/138526838/21f97ee9-b8fe-4fa9-b162-ba2e29456bfa)

**Comment by tteck** [comment, marked as answer]
It seems that you're facing a network problem, _probably_ linked to DNS.

**Comment by Schwobi** [comment]
Wow stupid. - of course. I blocked access to the internet a while ago. Thank you :)
